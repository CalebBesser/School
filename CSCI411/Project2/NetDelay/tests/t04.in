28
addEdge 0 3 31
addEdge 1 2 13
addEdge 2 3 28
addEdge 2 4 29
addEdge 4 6 12
addEdge 4 26 28
addEdge 5 6 16
addEdge 5 11 24
addEdge 6 9 14
addEdge 7 8 28
addEdge 7 9 20
addEdge 7 10 29
addEdge 7 20 30
addEdge 7 21 35
addEdge 7 22 17
addEdge 11 13 12
addEdge 11 14 5
addEdge 11 23 10
addEdge 12 13 18
addEdge 14 15 7
addEdge 16 23 21
addEdge 17 23 22
addEdge 18 23 24
addEdge 19 23 15
addEdge 24 27 23
addEdge 25 26 32
addEdge 25 27 27
printGraphW
findNetDelay
