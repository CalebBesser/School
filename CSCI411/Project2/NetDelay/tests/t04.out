Graph is:
0: 3/31	
1: 2/13	
2: 1/13	3/28	4/29	
3: 0/31	2/28	
4: 2/29	6/12	26/28	
5: 6/16	11/24	
6: 4/12	5/16	9/14	
7: 8/28	9/20	10/29	20/30	21/35	22/17	
8: 7/28	
9: 6/14	7/20	
10: 7/29	
11: 5/24	13/12	14/5	23/10	
12: 13/18	
13: 11/12	12/18	
14: 11/5	15/7	
15: 14/7	
16: 23/21	
17: 23/22	
18: 23/24	
19: 23/15	
20: 7/30	
21: 7/35	
22: 7/17	
23: 11/10	16/21	17/22	18/24	19/15	
24: 27/23	
25: 26/32	27/27	
26: 4/28	25/32	
27: 24/23	25/27	

The longest path is:
0 3 2 4 26 25 27 24 
The length of the longest path is 198.
